Parser/pegen/pegen.c

```diff
@@ -1118,8 +1137,8 @@ star_etc(Parser *p, arg_ty vararg, asdl_seq *kwonlyargs, arg_ty kwarg)
     return a;
 }
 
-static asdl_seq *
-_join_seqs(Parser *p, asdl_seq *a, asdl_seq *b)
+asdl_seq *
+join_sequences(Parser *p, asdl_seq *a, asdl_seq *b)
```

Comment:
This reminds me that we have a lot of non-static functions whose name doesn't start with `_Py`. Shouldn't we rename those? I believe it's still a strict rule to use `_Py` for public functions that aren't part of the public API.

Maybe this is something we'll need to clean up later?

Reply:
Yeah, very likely we need some renaming and maybe to use some private API macros. As long as `make smelly` does not complain, we should be good.

Reply:
It does, 64 times!

Reply:
😅

Reply:
And will suggest leaving it for the end as this is likely just some renaming + surrounding with API macros.